src/ambianic/pipeline/ai/fall_detect.py

```diff
@@ -261,17 +311,20 @@ def fall_detect(self, image=None):
                 elif not self.is_body_line_motion_downward(left_angle_with_yaxis, rigth_angle_with_yaxis):
                     log.info("The body-line angle with vertical axis is decreasing from the previous frame.")
                 else:
-                    angle_change = self.find_changes_in_angle(pose_dix)
+                    leaning_angle = self.find_changes_in_angle(pose_dix)
 
-                    if angle_change > self._fall_factor:
+                    leaning_probability = 1 if leaning_angle > self._fall_factor else 0
+                    fall_score = leaning_probability * (self.previous_body_vector_score + current_body_vector_score) / 2
+                    
+                    if leaning_angle > self._fall_factor:
```

Comment:
@bhavikapanara this is perfectly fine, but technically for clarity we should probably use
`if fall_score >= confidence_threshold` which does the same as the current check `leaning_angle > self._fall_factor` but is a bit easier to read.